pages/projects/[slug].tsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react'
 import Head from 'next/head'
+import { useRouter } from 'next/router'
 import ProjectPage from 'components/Projects/ProjectPage'
+import projects from 'data/projects.json'
 
-const Project = () => (
-  <>
-    <Head>
-      <title>Project: Coders for Causes</title>
-      <meta
-        name='description'
-        content='Coders for Causes is a not for profit organisation that empowers charities and other not for profit organisations by connecting them with university students to develop technical solutions. We are a student-run club based in Perth, Western Australia with a wide range of clients. Whether you are looking for technical advice or a long term project, get in touch with us for more information.'
-      />
-    </Head>
-    <ProjectPage />
-  </>
-)
+const Project = () => {
+  const router = useRouter()
+  const { slug } = router.query
+  const project = projects.find((project) => project.id === slug)
```

Comment:
i think prettier is set up to not use paran for single param, so this might be mistake